panic: runtime error: invalid memory address or nil pointer dereference

Today, I tried volt but failed with the following messages.

```
INSERT 04:13 : volt get lambdalisue/gina.vim                                                                                  ~
[INFO] Full building /Users/alisue/.vim/pack/volt/opt directory ...
[INFO] Installing vimrc and gvimrc ...
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x1068090]

goroutine 1 [running]:
syscall.sendfile(0xb, 0xa, 0x0, 0x4930, 0xc42000e100, 0x0, 0x0)
        /usr/local/Cellar/go/1.9.2/libexec/src/syscall/syscall_darwin_amd64.go:56 +0x40
syscall.Sendfile(0xb, 0xa, 0x0, 0x4930, 0xc42000e150, 0x0, 0x0)
        /usr/local/Cellar/go/1.9.2/libexec/src/syscall/syscall_unix.go:328 +0x49
github.com/vim-volt/volt/fileutil.CopyFile(0xc42034ec20, 0x18, 0xc42034ec80, 0x1c, 0xc4208d8000, 0x4930, 0x4930, 0x1a4, 0xc42034ec40, 0x19)
        /Users/alisue/.go/src/github.com/vim-volt/volt/fileutil/copyfile_unix.go:46 +0x3e4
github.com/vim-volt/volt/cmd.(*buildCmd).installVimrcAndGvimrc(0x174f850, 0xc4202409b2, 0x7, 0xc42034ec20, 0x18, 0xc42034ec40, 0x19, 0xc420080101, 0x0, 0x0)
        /Users/alisue/.go/src/github.com/vim-volt/volt/cmd/build.go:345 +0x59d
github.com/vim-volt/volt/cmd.(*buildCmd).doBuild(0x174f850, 0x174f801, 0x0, 0xe)
        /Users/alisue/.go/src/github.com/vim-volt/volt/cmd/build.go:267 +0x659
github.com/vim-volt/volt/cmd.(*getCmd).doGet(0x174f850, 0xc420134200, 0x1, 0x20, 0x174f640, 0xc420099e50, 0x0, 0x0)
        /Users/alisue/.go/src/github.com/vim-volt/volt/cmd/get.go:251 +0x6f5
github.com/vim-volt/volt/cmd.Get(0xc4200820b0, 0x1, 0x1, 0x3)
        /Users/alisue/.go/src/github.com/vim-volt/volt/cmd/get.go:146 +0x191
main.Main(0x13af72e)
        /Users/alisue/.go/src/github.com/vim-volt/volt/main.go:23 +0x1d6
main.main()
        /Users/alisue/.go/src/github.com/vim-volt/volt/main.go:11 +0x22
INSERT 04:14 [!] 3 : volt version                                                                                             ~
volt version: v0.2.0
INSERT 04:14 : 
```

## Info

I installed volt by `go get` and my OS is macOS High Sierra (10.13.2).
My terminal is alacritty 9bdac6 and my shell is zsh 5.4.2 on tmux 2c6af06.
My go is 1.9.2 installed.

--- a/.travis.yml
+++ b/.travis.yml
@@ -1,8 +1,17 @@
 language: go
 
-go:
-  - 1.9.1
-  - tip
+matrix:
+  include:
+    - os: linux
+      go: 1.9.1
+    - os: linux
+      go: tip
+    - os: osx
+      osx_image: xcode8.3
+      go: 1.9.1
+    - os: osx
+      osx_image: xcode8.3
+      go: tip
 
 script:
   - go vet -v ./...

--- a/fileutil/copyfile_generic.go
+++ b/fileutil/copyfile_generic.go
@@ -1,4 +1,4 @@
-// +build !darwin,!dragonfly,!freebsd,!linux,!netbsd,!openbsd,!solaris
+// +build !linux
 
 package fileutil
 

--- a/fileutil/copyfile_linux.go
+++ b/fileutil/copyfile_linux.go
@@ -1,4 +1,4 @@
-// +build darwin dragonfly freebsd linux netbsd openbsd solaris
+// +build linux
 
 package fileutil
 

